spark/setup.cfg

```diff
@@ -0,0 +1,10 @@
+[aliases]
```

Comment:
we're getting rid of this file and moving it into the tox.ini file. Can you remove it and move it there. If you look at the redis check it does that